src/Model/ValueObject/Identity/UuidFactory/UuidFactoryInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Model\ValueObject\Identity\UuidFactory;
+
+use CultuurNet\UDB3\Model\ValueObject\Identity\Uuid;
+
+interface UuidFactoryInterface
```

Comment:
This can become `UuidFactory` and then you have 2 flavours `FixedUuidFactory` and `GeneratedUuidFactory`. That means I would also rename `UuidGenerationFactory` to `GeneratedUuidFactory`